test/custom_class.ll

```diff
@@ -0,0 +1,100 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc
+
+; --- Source code ---
+; Generated with "clang++ -emit-llvm -S"
```

Comment:
`clang++ -emit-llvm -S` won't produce `spir64` triple, and for sure it won't produce `sycldevice` environment in the triple.

I guess that you used `clang++ -fsycl -fsycl-device-only -emit-llvm -S` or something like this. Also it is probably better to state that you used clang from intel/llvm project and not from llvm/llvm-project